python-sdk/src/astro/databases/snowflake.py

```diff
@@ -766,6 +818,68 @@ def _build_merge_sql(
         }
         return statement, params
 
+    def append_table(
+        self,
+        source_table: BaseTable,
+        target_table: BaseTable,
+        source_to_target_columns_map: dict[str, str],
+    ) -> None:
+        """
+        Append the source table rows into a destination table.
+        The argument `if_conflicts` allows the user to define how to handle conflicts.
+
+        Overriding the base method since we need to add quotes around the identifiers for
+         snowflake to preserver case of cols - Column(name=col, quote=True)
+
+        :param source_table: Contains the rows to be appended to the target_table
+        :param target_table: Contains the destination table in which the rows will be appended
+        :param source_to_target_columns_map: Dict of source_table columns names to target_table columns names
+        """
+        target_table_sqla = self.get_sqla_table(target_table)
+        source_table_sqla = self.get_sqla_table(source_table)
+        use_quotes_target_table = SnowflakeDatabase.use_quotes(target_table_sqla.columns.keys())
+        use_quotes_source_table = SnowflakeDatabase.use_quotes(source_table_sqla.columns.keys())
+        target_columns: list[column]
+        source_columns: list[column]
+
+        if not source_to_target_columns_map:
+            target_columns = [
+                Column(name=col.name, quote=use_quotes_target_table, type_=col.type)
+                for col in target_table_sqla.c.values()
+            ]
+            source_columns = target_columns
+        else:
+            # We are adding the VARCHAR in Column(name=col, quote=True, type_=VARCHAR) as a placeholder since the
+            # Column object requires it. It has no effect on the final query generated.
+            target_columns = [
+                Column(name=col, quote=use_quotes_target_table, type_=VARCHAR)
+                for col in source_to_target_columns_map.keys()
+            ]
+            source_columns = [
+                Column(name=col, quote=use_quotes_source_table, type_=VARCHAR)
+                for col in source_to_target_columns_map.keys()
+            ]
+
+        sel = select(source_columns).select_from(source_table_sqla)
+        # TODO: We should fix the following Type Error
+        # incompatible type List[ColumnClause[<nothing>]]; expected List[Column[Any]]
+        sql = insert(target_table_sqla).from_select(target_columns, sel)  # type: ignore[arg-type]
+        self.run_sql(sql=sql)
+
+    @staticmethod
+    def get_merge_initialization_query(parameters: tuple[str]) -> str:
+        """
+        Handles database-specific logic to handle constraints, keeping
+        it agnostic to database.
+        """
+        identifier_enclosure = ""
+        if SnowflakeDatabase.use_quotes(parameters):
```

Comment:
```suggestion
        if cls.use_quotes(parameters):
```

if you make this https://github.com/astronomer/astro-sdk/pull/1050/files#r998794279 change

Reply:
Done